lua/treesitter-context.lua

```diff
@@ -260,6 +262,7 @@ M.enable()
 
 api.nvim_command('command! TSContextEnable  lua require("treesitter-context").enable()')
 api.nvim_command('command! TSContextDisable lua require("treesitter-context").disable()')
+vim.cmd('highlight default link TreesitterContext NormalFloat')
```

Comment:
One small adjustment here, use `api.nvim_command` like the above instead of `vim.cmd`. Not sure which one is best but I'd rather it be consistent.

Reply:
I pushed a fixup commit for it.